content/hi/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+---
+title: पॉड्स और नोड्स देखना
```

Comment:
Are we exploring Pods & Nodes or seeing Pods & Nodes?

Reply:
The title in English version says `Viewing Pods and Nodes`. So, I've gone with `पॉड्स और नोड्स देखना`

Reply:
cc @divya-mohan0209

Reply:
I understand that, but dekhna sounds like we are just looking at pods and nodes. Would जांच करना be more appropriate?

Reply:
Yes, जांच करना would be better